src/scripts.js

```diff
@@ -103,9 +112,16 @@ function showHomePage() {
   show(searchContainer);
 }
 
+function showFavoritesPage() {
+  hide(homePage);
+  hide(searchContainer);
+  hide(recipePage);
+  show(favoritesPage);
+}
+
 function viewRecipe(event) {
   selectedRecipeName = event.target.innerText;
-  selectedRecipe = allRecipes.filter(recipe => selectedRecipeName === recipe.name);
+  selectedRecipe = allRecipes.filter(recipe => selectedRecipeName === recipe.name)[0];
```

Comment:
Great job - figuring out how to access index[0] to prevent repeats when adding to array.